packages/cursorless-engine/src/languages/latex.ts

```diff
@@ -191,6 +212,10 @@ const nodeMatchers: Partial<
   subParagraph: "subparagraph",
 
   environment: ENVIRONMENTS,
+  xmlElement: ENVIRONMENTS,
```

Comment:
seemed consistent to support this if we're supporting all the other xml scopes related to elements